lib/client/websocket-tracker.js

```diff
@@ -299,11 +300,11 @@ WebSocketTracker.prototype._startReconnectTimer = function () {
   var ms = Math.floor(Math.random() * RECONNECT_VARIANCE) + Math.min(Math.pow(2, self.retries) * RECONNECT_MINIMUM, RECONNECT_MAXIMUM)
 
   self.reconnecting = true
-  var reconnectTimer = setTimeout(function () {
+  self.reconnectTimer = setTimeout(function () {
```

Comment:
Add `self.reconnectTimer = null` to the constructor, as a form of "documentation".